lumispy/signals/luminescence_spectrum.py

```diff
@@ -344,6 +345,45 @@ def remove_background_from_file(self, background=None, inplace=False, **kwargs):
                 return self.map(lambda s, bkg: s - bkg, bkg=bkg_y, inplace=True)
 
 
+    SAVETXT_EXAMPLE = \
+    """
+    Examples
+    --------
+    >>> import lumispy as lum
+    >>> import numpy as np
+    
+    # Spectrum:
+    >>> s = lum.signals.LumiSpectrum(np.arange(5))
+    >>> s.savetxt('spectrum.txt')
+    # 0.00000	0.00000
+    # 1.00000	1.00000
+    # 2.00000	2.00000
+    # 3.00000	3.00000
+    # 4.00000	4.00000
+    
+    # Linescan:
+    >>> l = lum.signals.LumiSpectrum(np.arange(25).reshape((5,5)))
+    >>> l.savetxt('linescan.txt')
+    # 0.00000	0.00000	1.00000	2.00000	3.00000	4.00000
+    # 0.00000	0.00000	5.00000	10.00000	15.00000	20.00000
+    # 1.00000	1.00000	6.00000	11.00000	16.00000	21.00000
+    # 2.00000	2.00000	7.00000	12.00000	17.00000	22.00000
+    # 3.00000	3.00000	8.00000	13.00000	18.00000	23.00000
+    # 4.00000	4.00000	9.00000	14.00000	19.00000	24.00000
+    """
+
+    def savetxt(self, filename, fmt='%.5f', delimiter='\t', axes=True,
+                transpose=False, **kwargs):
+        """Writes luminescence spectrum object to simple text file.
+        %s
+        %s
+        %s
+        """
+        savetxt(self, filename, fmt, delimiter, axes, transpose, **kwargs)
```

Comment:
I think these function should be called slightly different. I find it confusing that the utils and the LumiSpectrum function have the same name...

Reply:
Well, they do exactly the same - except that the utils-functions also accepts other hyperspy signal classes than LumiSpectrum. The latter is more for convenience. So I woul prefer keeping the same name.